src/util.ts

```diff
@@ -43,19 +45,21 @@ export function parseAzureResourceId(resourceId: string): { [key: string]: strin
 }
 
 export async function showWorkspaceFoldersQuickPick(placeHolderString: string, telemetryProperties: TelemetryProperties): Promise<string> {
-    const browse: IQuickPickItemWithData<vscode.WorkspaceFolder> = { label: '$(file-directory) Browse...', description: '', data: undefined };
-    const folderQuickPickItems = vscode.workspace.workspaceFolders ?
-        vscode.workspace.workspaceFolders.map((value) => {
-            {
-                return <IQuickPickItemWithData<vscode.WorkspaceFolder>>{
-                    label: value.name,
-                    description: value.uri.fsPath,
-                    data: value
-                };
-            }
-        }) :
-        [];
-    folderQuickPickItems.unshift(browse);
+    const folderQuickPickItems = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders.map((value) => {
+        {
+            const subpath: string = vscode.workspace.getConfiguration(extensionPrefix).get(configurationSettings.deploySubpath);
```

Comment:
So currently we only use `showWorkspaceFoldersQuickPick` for deploying in this repo, but it's a generic function that we could use in other places. In those cases, we don't want to add `deploySubpath`. Please add an optional parameter like 'subPathSetting' where you pass in configurationSettings.deploySubpath

Reply:
I attempted merging your master over mine (following [this guide](https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/), doesn't look like it effected the whitespace in that file though. I did merge your master over to mine prior to working on this as well, which has me wondering if that's not where the whitespace changes came from in the first place...

Reply:
`subPathSetting` param added to `showWorkspaceFoldersQuickPick`.